contracts/SignatureAccount.sol

```diff
@@ -31,10 +34,13 @@ contract SignatureAccount is IAccount {
         //
         // HINT: OpenZeppelin `ECDSA` library has a `tryRecover(hash, signature)` function to recover the signature signer.
 
+        (bool success, ) = payable(entryPoint).call{ value: missingAccountFunds }("");
+        (success);
         return SIG_VALIDATION_SUCCEEDED;
     }
 
     function execute(address target, uint256 value, bytes calldata data) external {
+        require(msg.sender == entryPoint, "SignatureAccount: Unauthorized caller");
```

Comment:
Do we want to add this to every account? The `DepositAccount` seems missing. Or maybe we should extract a base account with this `execute` function for all accounts.

Reply:
I cover this check in SignatureAccount section so I added it to the example accounts thereafter.